contents/handbook/growth/marketing/messaging_framework.md

```diff
@@ -0,0 +1,87 @@
+
+- Combine quantitative and qualitative data about product usage.
+- Adopt a more experimental approach to product development.
+- Use open-source software, potentially with an eye towards customization.
+- Move forward quickly with immediate deployment. 
+
+## Positioning statement
+Our positioning statement is written using [Geoffrey Moore’s template](https://gist.github.com/JoshSmith/2041454). Our positioning statement is:
+
+_For product managers who need to understand their users, PostHog is the only open-source, all-in-one analytics platform that offers everything from funnel analysis and cohort tracking, to feature flags and session recording._ 
+
+_Unlike traditional tools, PostHog offers self-hosted deployments so that data doesn't need to be shared with anyone -- not even PostHog._
+
+## Alternative
+There are some alternative software which PostHog's users may consider. We split these into strong (e.g. regularly cited, competitive on multiple features) and weak (e.g. irregularly cited, competitive on few features). 
+
+_Strong:_ Heap Analytics, MixPanel, Amplitude, Pendo
+
+_Weak:_ HotJar (session recording), Optimizely (experimentation), Smartlook (analytics), FullStory (session recording & analytics), KissMetrics (analytics)
+
+## Messaging pillars
+Messaging pillars are our three core benefits or points of differentiation; points that define our product and our positioning; ideas which we own fully and reinforce regularly. The niches we dominate. 
+
+**All-in-one** — PostHog combines multiple types of analytics tools
+**Self-hosted** — PostHog can be deployed onto a clients’ own infrastructure
+**Extensible** — PostHog’s value can be extended beyond core metrics
+
+It may be tempting to ask of the second point: why we don’t describe this as ‘hosting flexibility’ or ‘hosting options’ given that PostHog can also be deployed on a cloud? The answer is that many alternative soplutions can be deployed on a cloud. Only PostHog can be self-hosted. 
+
+## Supporting messages
+Supporting messages are the key features/benefits which back up each pillar and can be used to prove its validity to customers. Where possible, these should tie to hard data. 
+
+**1 - All-in-one**
+_Analyze:_ Auto-capture events and analyze retroactively through funnel analysis, retention tables, path tracking, and more. 
+_Experiment:_ Test ideas with feature flags that target specific users, cohorts, or selected groups based on personas. 
+_Solve:_ Dive into session recordings to identify bugs or contextualize quantitative data with new perspectives. 
+
+**2 - Self-hosted**
+_Insights:_ Overcome ad-blockers that detect third-party cookies, giving you better data.
+_Private:_ Deploy on your infrastructure and don’t share data with anyone. Not even us.
+_Control:_ Control everything, from when to upgrade to how and where to deploy. 
+
+**3 - Extensible**
+_Open Source:_ Contribute to our code and join the 10k+ developers in our community 
```

Comment:
Where is the 10k figure from?

Reply:
https://github.com/PostHog/posthog.com/pull/1535#discussion_r660463574